scripts/release.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd ./packages/app
+
+# Clean up old build artefacts
+rm -rf ./build/app/outputs/flutter-apk
+
+# Make a build per architecture
+flutter build apk \
+  --flavor normal \
+  --split-per-abi \
+  --release \
+  --obfuscate \
+  --split-debug-info ./build/app/outputs/flutter-apk
+
+# Make a bundle for all architectures
+flutter build apk \
+  --flavor normal \
+  --release \
+  --obfuscate \
+  --split-debug-info ./build/app/outputs/flutter-apk
```

Comment:
This runs the compiler once to build apks for each architecture, they're nice and small then (~15mb), and then one "bundled" apk for all architectures (~50mb)

Reply:
Don't know if this is necessary or if we should just go with the bundled one?